tests/test_addons.py

```diff
@@ -339,7 +339,7 @@ def test_get_or_start_returns_error(self):
         class MyException(exceptions.AddonEnrichmentError):
 
             @property
-            def renderable_error(self):
+            def render_error(self):
```

Comment:
Rule of thumb: properties are nouns, methods are nouns. If you rename `renderable_error` to `as_html` as suggested in a previous comment, you can change this back to `renderable_error`.